src/components/Resume/Resume.js

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import "./Resume.css";
+
+export default function Resume() {
+    return (
+        <div>
+            <div className="title">
+                <picture>
+                    <img src="./avatar.jpg" alt="avatar" />
+                </picture>
+                <div>
+                    <h1>MOHAMAD SOUFI</h1>
+                    <h3>FULL-STACK DEVELOPER</h3>
+                </div>
+            </div>
+            <div className="content-container">
+                <div className="left-side">
+                    <h2>PERSONAL PROFILE</h2>
+                    <p>
+                        I am a full stack web developer with a strong passion
+                        for
+                        <br /> coding , I started as a self-taught , and after a
+                        couple of
+                        <br /> online courses and weekend workshops I fully
+                        committed to
+                        <br /> learning and joined an on-site intensive bootcamp
+                        to
+                        <br /> get more familiar with the cutting edge
+                        technologies
+                        <br /> and industry standards. I am eager to be a part
+                        of a
+                        <br /> motivated team to bring fresh ideas and grow
+                        together.
+                    </p>
+
+                    <br />
+
+                    <h2>CONTACT DETAILS</h2>
+
+                    <p>
+                        mail: mohamad.soufi@outlook.com
+                        <br /> Mobile: +49 157-864-33517 <br /> Adress: Berlin,
+                        Germany
+                        <br /> Social:
+                    </p>
+                    <br />
+
+                    <h2>LANGUAGES</h2>
+
+                    <p>
+                        Native: Arabic
+                        <br /> Fluent: English
+                        <br /> Knowledgeable: German
+                    </p>
+
+                    <br />
+
+                    <h2>STACK</h2>
+
+                    <ul>
+                        <li>React.js, Redux, Vue.js, jQuery, Flexbox, Grid</li>
+                        <li>
+                            Node.js, Express, PostgreSQL, REST, web sockets,{" "}
+                            <br />
+                            Handlebars
+                        </li>
+                        <li>Jest, React Testing Library</li>
+                        <li>Git,GitHub</li>
+                        <li>AWS , Heroku</li>
+                    </ul>
+                </div>
+                <div className="right-side">
+                    <h2>EMPLOYMENT HISTORY</h2>
+                    <br />
+                    <h4>RIDER</h4>
+                    <p>Foodora.Co | 2018 - 2019</p>
+                    <br />
+                    <h2>VOLUNTARY EXPERIENCE</h2>
+                    <h4>DRIVER, WAITER, AND HR</h4>
```

Comment:
try to write a couple of sentences here and better if you break the titles down .. like : Driver : transporting donations from donors to the storage facility 
...etc